WeatherXM/wxm-android
# FE-980 Integrate cell center in public devices

## **Why?**
Integrate `cellCenter` in the Device of the cell model and ensure tapping on address works

### **How?**
Just added the `cellCenter` field in the API Model and in the transformation to the `UIDevice` model.

### **Testing**
1. Open explorer
2. Go to a random device in a cell (not owned or followed)
3. Open its device details
4. Click on the address cell and ensure that everything works
